blueprints/mocha/ember-cli-yadda/files/tests/helpers/yadda.js

```diff
@@ -1,2 +1,2 @@
-import yadda from 'npm:yadda';
+import yadda from 'yadda';
```

Comment:
As these files are part of the blueprint, they won't get updated automatically in a user's app. What would happen if `npm:` is unchanged? Would that break things, i.e. the changes here are breaking changes that require a major version bump?

And what's actually the point of having this file still around, as it is just reexporting the unchanged import?